.env.example

```diff
@@ -12,4 +12,9 @@ AUTH_PEPPER=secretPepper
 
 # DB Options
 DB_URI=yourMongoUri
-TEST_DB_URI=yourTestMongoUri
\ No newline at end of file
+TEST_DB_URI=yourTestMongoUri
+
+#Google auth
+# Todo: Change to production credentials
+GOOGLE_CLIENT_ID= 114619963004-2mec4e5gicsjgn83ufq0b9cudd0apgkp.apps.googleusercontent.com 
```

Comment:
Make sure you integrate these new env variables into our config system

Reply:
I did? my bad, forgot to delete the lock file after the installation (was lazy to search for the global install in yarn 😄). 

I will add those to the config system, and sure, later on we can cross-reference it